mock-thunderhead/src/main/resources/application.yml

```diff
@@ -43,7 +43,7 @@ auth:
       efs.enable: false
       customimage.enable: true
       loadbalancer.enable: true
-      backup.on.upgrade.enable: false
+      backup.on.upgrade.enable: true
```

Comment:
[@sgarcia.acosta] backup.on.upgrade.enable in application.yaml.
I remember this will fail because of some limitations in the e2e infra. Can you confirm it?

Reply:
I don't see the answer for this one

Reply:
@sxxgrc Do you mind putting the answer there? I remember we talked about this during standup and the answer was this should have been marked as "true" long time ago but not sure why it is still "false"? Thank you so mmuch!

Reply:
As per an internal conversation, this configuration is used for mocks tests but by the E2E tests executed for promoting the changes to mow-dev.

Reply:
This shouldn't cause any failures, only local developments and integration tests (mock tests) run during PRs use the local versions of `application.yml` with these variables being handled separately for e2e tests. This was something informed to me by members of the CB team a few months ago.

As far as I know there is no reason for this not to be enabled. Even if this somehow affects e2e tests, it would just mean that a backup would be run during the upgrade process, which would just communicate with the mock DR service and still be successful.

Reply:
ok, we can merge this, but:
- next time please highlight in the commit message that you are turning on something for e2e
- we will have to revert the whole change if this breaks e2e